seqr/views/apis/anvil_workspace_api_tests.py

```diff
@@ -212,12 +227,12 @@ def test_create_project_from_workspace(self, mock_file_iter, mock_file_exist, mo
         self.assertEqual(response.reason_phrase, 'Project "{name}" for workspace "{namespace}/{name}" exists.'
                          .format(namespace=TEST_WORKSPACE_NAMESPACE, name=TEST_NO_PROJECT_WORKSPACE_NAME))
 
-        # Test sending email exception
+        # Test sending slack message exception
         url = reverse(create_project_from_workspace, args=[TEST_WORKSPACE_NAMESPACE, TEST_NO_PROJECT_WORKSPACE_NAME2])
-        mock_email.side_effect = Exception('Something wrong while sending email.')
+        mock_slack.side_effect = Exception('Something wrong while sending the slack message.')
```

Comment:
If you are mocking `safe_post_to_slack` you don't need to test error handling, as that function handles its own exceptions. If you want to test exception handling you need to mock it at a lower level, mocking the redis client in the helper function itself. Or you can opt not to test error handling since this helper function is tested elsewhere. Its your call